package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Women's Healthcare resources in the palm of your hand.",
   "main": "server.js",
   "scripts": {
-    "start": "NODE_ENV=localhost node ./server/server.js",
+    "client-dev": "webpack-dev-server",
+    "start": "NODE_ENV=localhost nodemon ./server/server.js",
```

Comment:
By changing this to `nodemon`, it will watch the server files and automatically restart the server for you so you don't have to restart it manually. It's a SUPER useful tool for development! 😄 You can continue to run `npm start` to start the server. See the updated CONTRIBUTING doc for the commands to install `nodemon` globally.